_posts/2021/2021-03-19-the-x-factor.md

```diff
@@ -0,0 +1,59 @@
+---
+title: The X factor or how I learned to stop worrying and love the backend
+layout: post
+date: 2021-03-19
+header_image: public/micronaut-header.jpeg
+header_position: center
+header_overlay: true
+category: on-the-job
+tags: ["agile", "company", "team"]
+authors: ["Paolo"]
+about_authors: ["ppriotto"]
+---
+
+It was autumn 2020. I had been at ePages for almost six years, and always in the same frontend team. You could say I was ready for some change. So when our scrum masters started asking around who could imagine joining a new, cross functional team, I was immediately up for it. Four months later, I want to recapitulate the journey since then.
```

Comment:
A bit unclear, how long the new team exists. Maybe: "Four months ago, the new team started to work. Time to recapulate the journey since then."